lib/hexpm/accounts/audit_log.ex

```diff
@@ -138,6 +138,15 @@ defmodule Hexpm.Accounts.AuditLog do
   defp extract_params("password.reset.finish", nil), do: %{}
   defp extract_params("password.update", nil), do: %{}
 
+  defp extract_params("billing.create", params),
```

Comment:
I found that all the other `params` for `extract_params/2` are either a tuple or `nil`. Do we need to make this one a tuple as well?